src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -129,4 +129,68 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
 template <size_t SpatialDim, typename Frame, typename DataType>
 tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes spatial derivatives of lapse (N) from the generalized
```

Comment:
I prefer `\f$alpha\f$` and `\f$beta\f$` for lapse and shift, but I’ll defer if everyone else disagrees.

Reply:
Probably a philosophical debate, but we currently use both `N` and `alpha` in different parts of the code, so we should file an issue to make the code base consistent (whatever is decided on). Does that sound okay with you, @geoffrey4444 ?

Reply:
Yep!

Reply:
I actually prefer `N` and `\vec{N}` for the two, but this is a question for broader consensus..

Reply:
Issue opened: #1209